docs/quick_tour.rst

```diff
@@ -827,7 +819,7 @@ Pyramid and SQLAlchemy are great friends. That friendship includes a scaffold!
 
   $ pcreate --scaffold alchemy sqla_demo
   $ cd sqla_demo
-  $ python setup.py develop
+  $ env/bin/pip install -e .
```

Comment:
The quick_tour was kind of loosie goosie with the commands, so i went with the tried and true environment variable.